From 7f218e224a6f8a3770a0fc0f7304ecac4e2176a4 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Thu, 24 Oct 2019 14:15:42 +0100
Subject: [PATCH] OS-8033 smartos-only-release target should include tests-tar
 Reviewed by: John Levon <john.levon@joyent.com> Approved by: John Levon
 <john.levon@joyent.com>

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 54655f47..66f08cfa 100644
--- a/Makefile
+++ b/Makefile
@@ -681,6 +681,7 @@ triton-smartos-release: \
 
 .PHONY: smartos-only-release
 smartos-only-release: \
+    tests-tar \
     common-platform-publish \
     smartos-build \
     smartos-publish \
-- 
2.17.2 (Apple Git-113)

